vector/src/main/java/im/vector/activity/VectorRoomCreationActivity.java

```diff
@@ -135,7 +137,7 @@ public void OnRemoveParticipantClick(ParticipantAdapterItem item) {
             public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                 // the first one is "add a member"
                 if (0 == position) {
-                    launchInviteMembersActivity(RoomCreationModes.DISCUSSION , INVITE_USER_REQUEST_CODE);
+                    launchInviteMembersActivity(RoomCreationModes.DISCUSSION , VectorRoomInviteMembersActivity.ContactsFilter.NO_TCHAP_ONLY, INVITE_USER_REQUEST_CODE);
```

Comment:
We should consider the current mMode value before calling launchInviteMembersActivity.
(The case mMode == DIRECT_CHAT would be ignored here because it is unexpected)
````
switch (mMode) {
   case DISCUSSION:
      launchInviteMembersActivity(RoomCreationModes.DISCUSSION, VectorRoomInviteMembersActivity.ContactsFilter.ALL, INVITE_USER_REQUEST_CODE);
      break;
   case INVITE:
      launchInviteMembersActivity(RoomCreationModes.INVITE, VectorRoomInviteMembersActivity.ContactsFilter.NO_TCHAP_ONLY, INVITE_USER_REQUEST_CODE);
      break;
}
`````